go-apps/meepctl/cmd/deploy.go

```diff
@@ -211,11 +238,15 @@ func deployRunScriptsAndGetFlags(targetName string, chart string, cobraCmd *cobr
 		deploySetGrafanaValues(chart, cobraCmd)
 		flags = utils.HelmFlags(nil, "--set", "persistentVolume.location="+deployData.workdir+"/grafana/")
 		flags = utils.HelmFlags(flags, "--values", deployData.workdir+"/tmp/grafana-values.yaml")
+		altServer := utils.RepoCfg.GetBool("repo.deployment.alt-server")
+		flags = utils.HelmFlags(flags, "--set", "altIngress.enabled="+strconv.FormatBool(altServer))
 	case "meep-influxdb":
 		flags = utils.HelmFlags(nil, "--set", "persistence.location="+deployData.workdir+"/influxdb/")
```

Comment:
Influx is currently exposed using an ingress, however it is not used by grafana (Grafana is configured in proxy mode instead of direct DB access mode). We should consider disabling the influxDB ingress or adding an alt-ingress if we plan on using it.